src/public/js/_.js

```diff
@@ -758,14 +758,14 @@ module.exports = {
 
 		// create labels depending on chartPeriod, Screen size, groupBy
 		switch (chartPeriod) {
-			case 'week':
+			case 's-week':
```

Comment:
`week` and `s-week` are two different types, so this change is technically wrong in most places. Instead, this should simply be:

```
case 'week':
case 's-week':
    results.labels = this.createWeekPeriodChartLabels(results.labels);
    break;

Reply:
All these functions now expecting not value but type, since dropdown now has s-month type for all month-related values it is correct. I checked every case of usage, i think i didn't miss something.